main.tex

```diff
@@ -174,6 +163,8 @@ \section*{Introduction}
 %
 We use genotype and haplotype data derived from whole-genome Illumina sequencing of 765 individual mosquitoes collected from natural populations in 8 African countries to survey genetic diversity and study the evolutionary and demographic history of insecticide resistance at the \emph{Vgsc} locus.
 %
+Initial Ag1000g findings are confirmed then analyses extended to investigate the age, movement and recombination of insecticide resistance conferring haplotypes.
```

Comment:
I think we need to be a bit clearer here about the relationship with previously reported results. How about:

"Here we report an in-depth analysis of the \emph{Vgsc} locus using genotype and haplotype data from phase 1 of the Anopheles gambiae 1000 Genomes Project \cite{Ag1000G2016}.

These data are derived from whole-genome Illumina sequencing of 765 individual mosquitoes collected from natural populations in 8 African countries.

We presented some initial results regarding \emph{Vgsc} from analyses of these data in \cite{Ag1000G2016}, and here we describe a number of new analyses to confirm, extend and elaborate on our initial findings."